.github/workflows/generate-wiki-docs.yml

```
@@ -52,7 +52,7 @@ jobs:
       - name: Publish docs
         if: fromJSON(env.SHOULD_RUN)
         # @v3 https://github.com/Andrew-Chen-Wang/github-wiki-action/releases/tag/v3
-        uses: Andrew-Chen-Wang/github-wiki-action@194d37ff03f5220e1baf7eb9d7c7ad7f3e1db30c
+        uses: Andrew-Chen-Wang/github-wiki-action@50650fccf3a10f741995523cf9708c53cec8912a
```

This way Dependabot updates the version in the comment automatically:
```suggestion
        uses: Andrew-Chen-Wang/github-wiki-action@50650fccf3a10f741995523cf9708c53cec8912a # v4.4.0
```